cicecore/shared/ice_constants.F90

```diff
@@ -84,7 +84,8 @@ module ice_constants
         p027 = p055*p5, &
         eps04  = 1.0e-4_dbl_kind, &
         eps13  = 1.0e-13_dbl_kind, &
-        eps16  = 1.0e-16_dbl_kind
+        eps16  = 1.0e-16_dbl_kind, &
+        pi   = 3.14159265358979323846_dbl_kind
```

Comment:
I'm not in love with hardwiring pi here.  It's already hardwired into icepack.  What we really need is for pi to be computed at initialization or set by something like cesm shr_const depending on the situation.  Both icepack and cice could use the upgrade.  For now, we can leave it this way.

Reply:
Agree, I was surprised it wasn't in here to be honest. I'll add this as an issue. Maybe I can get to it before the release.

Reply:
Where the variables exist in icepack, we have tried to use those in CICE.  Both pi and rad_to_deg are available via queries to icepack.  That is probably the right way to do it.  Where you need rad_to_deg, query it in that subroutine from icepack and get rid of both pi and rad_to_deg definitions in cice.   I guess I am going to suggest that change at this point, should have said that earlier.

Reply:
Ok, I agree. I would rather do this the right way now. So I should definitely fix it. However, I have an additional question/clarification. 

It looks like pi is set in columnphysics/icepack_parameters.F90. Why not in /configuration/driver/icedrv_constants.F90?  It looks like we set numerical constants in both places, though the constants don't overlap. But I guess I don't understand why we don't just set them in one place. Am I being dense?

Reply:
You're certainly not being dense!  The idea with the constants being in two places in Icepack was to delineate the ones that are essential for the Icepack physics (for groups wanting to port it into their own sea ice models) from the ones that are only needed for the driver, e.g. for the forcing.  There might be some that are in the wrong place, and if you spot any, make an issue to fix them.  Thanks!